app/screens/client_upgrade/client_upgrade.js

```diff
@@ -24,10 +24,12 @@ import {popTopScreen, dismissModal} from 'app/actions/navigation';
 
 export default class ClientUpgrade extends PureComponent {
     static propTypes = {
-        actions: PropTypes.shape({
-            logError: PropTypes.func.isRequired,
+
+        /*actions: PropTypes.shape({
```

Comment:
Are these comments needed?

Reply:
I left them there as a reminder since the logError is semi-removed.

Reply:
I'll remove them though to clean this up

Reply:
please remove it if they are not needed, this will only generate confusion